support-frontend/assets/pages/paper-subscription-landing/components/hero/hero.jsx

```diff
@@ -27,6 +28,17 @@ function getHeading(): string {
 
 }
 
+const getDiscountCopy = () => {
+  const discountParam = getQueryParameter('discount');
```

Comment:
I think it would be good to change the name of the query param to something like `creative` or `copy` because if people see a param which is discount I imagine they might start playing around with it to see if they can get a bigger discount. I know I would!